src/main/kotlin/platform/mcp/aw/fixes/CopyAwAccessibleEntryFix.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2022 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.platform.mcp.aw.fixes
+
+import com.demonwav.mcdev.facet.MinecraftFacet
+import com.demonwav.mcdev.platform.fabric.FabricModuleType
+import com.demonwav.mcdev.platform.mcp.actions.CopyAwAction
+import com.demonwav.mcdev.util.findModule
+import com.intellij.codeInsight.daemon.QuickFixActionRegistrar
+import com.intellij.codeInsight.intention.IntentionAction
+import com.intellij.codeInsight.quickfix.UnresolvedReferenceQuickFixProvider
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiJavaCodeReferenceElement
+
+class CopyAwAccessibleEntryFix(val target: PsiElement, val element: PsiElement) : IntentionAction {
+
+    class Provider : UnresolvedReferenceQuickFixProvider<PsiJavaCodeReferenceElement>() {
+
+        override fun registerFixes(ref: PsiJavaCodeReferenceElement, registrar: QuickFixActionRegistrar) {
+            val module = ref.findModule() ?: return
+            val isFabric = MinecraftFacet.getInstance(module, FabricModuleType) != null
+            if (!isFabric) {
```

Comment:
I'm not sure about this, other platforms might use AWs outside of Fabric, Sponge for example.

I think the best solution would be to register the fix is an AW file exists in the module, but including some more platforms is enough to keep it simple.

Reply:
These changes don't require tracking a project-wide access widener, I left that for a future PR. What other platforms use AWs other than Sponge?

Reply:
I _think_ everything can use AWs if loaded early enough (similar to mixins), but spigot/bungee/velocity plugins are less likely to use them than on fabric/sponge (and forge ?)

Reply:
So just Sponge for now?

Reply:
Yeah it will be fine for now, thanks :)